File: 1_Year/CursoJava/salario_bonus.java
import java.util.Scanner;

public class salario_bonus {
    public static void main(String[] args) 
    {
    Scanner scanner = new Scanner(System.in);
    
    double money = scanner.nextDouble();
    double sell =scanner.nextDouble();
    scanner.close();
    // ingles pq é menorzinho
    money = money + sell*0.15;
    System.out.printf("TOTAL = R$ %.2f\n", money);
    }
}

File: 1_Year/AED l/www.beecrowd.com.br/Randon/salario_bonus.c
#include <stdio.h>

int main (){
    char name[20];
    double money, sell;
    // ingles pq é menorzinho
    scanf("%s", &name);
    scanf("%lf %lf", &money, &sell);
    money += sell*0.15;
    printf("TOTAL = R$ %.2lf\n", money);

    return 0;
}